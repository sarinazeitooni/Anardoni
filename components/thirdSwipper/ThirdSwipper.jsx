import { Swiper, SwiperSlide } from 'swiper/react';
import ThirdSwipperData from "../../data/ThirdSwipperData";
import { BrowserView, MobileView} from "react-device-detect";
const ThirdSwipper = ()=>{
    let data = ThirdSwipperData;
    return(
        <div className= "screenshot-container">
            <BrowserView>
                <Swiper spaceBetween={1} slidesPerView={4}>
                    {data.map((item) => <SwiperSlide>
                        <img className="screenshot" width="80%" height={350} src={item.url}/>
                    </SwiperSlide>)}
                </Swiper>
            </BrowserView>
            <MobileView>
                <Swiper spaceBetween={15} slidesPerView={3}>
                    {data.map((item) => <SwiperSlide>
                        <img className="screenshot" width={65} height={140} src={item.url}/>
                    </SwiperSlide>)}
                </Swiper>
            </MobileView>
        </div>
    )
};
export default ThirdSwipper;