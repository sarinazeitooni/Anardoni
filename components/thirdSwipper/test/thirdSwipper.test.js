import React from 'react';
import { shallow } from 'enzyme';
import ThirdSwipper from "../ThirdSwipper";
describe(" Third swiper component", () => {
    it("should render div as props length", () => {
        const swiper = shallow(<ThirdSwipper />);
        expect(swiper.find('Swiper').length).toBe(1);
        expect(swiper).toMatchSnapshot();
    });
    it("should render correctly" , ()=>{
        expect(shallow(<ThirdSwipper/>))
    })
});