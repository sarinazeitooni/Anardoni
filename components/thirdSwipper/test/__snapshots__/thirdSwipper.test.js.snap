// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Third swiper component should render div as props length 1`] = `ShallowWrapper {}`;
